### ISSUE TITLE
question about LM capcapacity, thanks

### ISSUE THREAD

**Issue by Michelleable** [soochan-lee/RoT#1]
Admire the work you do and say thanks for open source, but I have some questions.

After reading the paper, what I understand is that " To gain mathematical computing power, the model needs to be fine-tuned".
The capability is dramatically improved, but how about common ablity?

For example, A model like LLaMa or alpaca, even someone else after RLHF. After the training, how much impact does it have on the ability to generate models in other aspects? (such as chat, instructed ability) 

Is there any evaluation of the above mentioned？ thank you very much :)

**Comment by soochan-lee** [comment]
Thank you for your interest in our work, and sorry for the late reply.

First of all, "To gain mathematical computing power, the model needs to be fine-tuned" is far from the point of our paper.
We are proposing RoT as an _inference_ framework, which demonstrates the importance of utilizing multiple contexts to solve complex reasoning problems.
Fine-tuning is not the only way to make an LM perform RoT.
For example, one may prepend a CoT prompting, which includes several demonstrations of RoT, to every context to induce an LM to perform RoT without any fine-tuning.
However, as we deal with extremely complex problems that require hundreds of thousands of tokens of reasoning steps, each token generation should be extremely accurate to achieve a reasonable score.
We found that CoT prompting is not sufficient for such accuracy, so we chose to rely on fine-tuning for this work.

Of course, fine-tuning for solving just a few types of problems will damage the general capability of LMs.
We did not quantitatively measure the degradation because (i) it seemed obvious, and (ii) evaluating the general capability of LMs is costly.

Hope this answers your question :)
